frame/contracts/src/weights.rs

```diff
@@ -1316,36 +1318,36 @@ impl WeightInfo for () {
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: unknown [0x7afa01283080ef247df84e0ba38ea5a587d25ce6633a6bfbba02068c14023441] (r:1 w:1)
 	fn seal_set_storage_per_kb(n: u32, ) -> Weight {
-		(727_394_000 as Weight)
-			// Standard Error: 247_000
-			.saturating_add((73_350_000 as Weight).saturating_mul(n as Weight))
+		(346_704_000 as Weight)
+			// Standard Error: 200_000
+			.saturating_add((29_907_000 as Weight).saturating_mul(n as Weight))
 			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	fn seal_clear_storage(r: u32, ) -> Weight {
-		(163_262_000 as Weight)
-			// Standard Error: 1_492_000
-			.saturating_add((799_087_000 as Weight).saturating_mul(r as Weight))
+		(0 as Weight)
+			// Standard Error: 1_963_000
+			.saturating_add((746_581_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
 			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
 			.saturating_add(RocksDbWeight::get().writes((100 as Weight).saturating_mul(r as Weight)))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	fn seal_get_storage(r: u32, ) -> Weight {
-		(288_728_000 as Weight)
-			// Standard Error: 752_000
-			.saturating_add((434_780_000 as Weight).saturating_mul(r as Weight))
+		(69_873_000 as Weight)
+			// Standard Error: 722_000
+			.saturating_add((338_292_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	fn seal_contains_storage(r: u32, ) -> Weight {
-		(209_606_000 as Weight)
-			// Standard Error: 1_137_000
-			.saturating_add((680_187_000 as Weight).saturating_mul(r as Weight))
+		(0 as Weight)
+			// Standard Error: 1_169_000
+			.saturating_add((480_500_000 as Weight).saturating_mul(r as Weight))
```

Comment:
This is fine. We are only interested in what is multiplied with `r`.

Reply:
have you graphed this output?

The standard error is VERY large here, which implies a very non-linear graph

Reply:
Looks large at first, but the `r` parameter scales with *480 million* for each increment.  
So comparing that to the *1.2 million* error is fine I think. We could normalize this `standard error` to something relative to make it more comparable.

Computational weight without DB operations:  
![Figure_1](https://user-images.githubusercontent.com/10380170/150525250-c409cc41-f9ab-44ba-ab82-ff4dc63cd78c.png)

Reply:
Compared to the value itself it doesn't look large to me.